.changeset/witty-planes-serve.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-query': patch
+---
+
+Display subclasses in the explorer tree
```

Comment:
We can say: `Display subclasses in explorer tree, also add support DnD for these nodes to help build fetch structure`